aldryn_reversion/tests/base.py

```diff
@@ -102,3 +107,57 @@ def revert_to(self, object_with_revision, revision_number):
             reversion.get_for_object(
                 object_with_revision)))[revision_number - 1]
         version.revision.revert()
+
+
+class HelperModelsObjectsSetupMixin(object):
+
+    def setUp(self):
+        super(HelperModelsObjectsSetupMixin, self).setUp()
+        # prepare objects that can be used with utility methods
+        # simple model
+        self.simple_registered = self.create_with_revision(
+            SimpleRegistered,
+            **{'position': 1})
+
+        self.simple_no_admin = self.create_with_revision(
+            SimpleNoAdmin,
+            **{'position': 42})
```

Comment:
Why are you writing it like this? The only reason to use `**` is if you have a dict variable. Since you know in advance the values, you could just write it thus:
```
self.simple_no_admin = self.create_with_revision(
    SimpleNoAdmin, position=42)
```